### PR TITLE
[MM-25646] Adds deleteall user command

### PR DESCRIPTION
**Pull request by mgdelacroix**
#### Summary
Adds the `mmctl --local user deleteall` command and a wrapper that makes it only runnable in local mode.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-25646

#### Related Pull Requests
- [Has server changes](https://github.com/mattermost/mattermost-server/pull/14903)
- [Has API docs changes](https://github.com/mattermost/mattermost-api-reference/pull/534)

### CODE REVIEW

File: commands/user.go [hunk 1]
```diff
@@ -84,6 +84,16 @@ If MFA enforcement is enabled, the user will be forced to re-enable MFA as soon
 	RunE:    withClient(resetUserMfaCmdF),
 }
 
+var DeleteAllUsersCmd = &cobra.Command{
+	Use:     "deleteall",
```
Comment (isacikgoz) [inline commands/user.go:88]: Should we use delete-all instead?

File: commands/user.go [hunk 2]
```diff
@@ -329,6 +342,32 @@ func resetUserMfaCmdF(c client.Client, cmd *cobra.Command, args []string) error
...
 }
 
+func deleteAllUsersCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	confirmFlag, _ := cmd.Flags().GetBool("confirm")
+	if !confirmFlag {
+		var confirm string
+		fmt.Println("Have you performed a database backup? (YES/NO): ")
+		fmt.Scanln(&confirm)
+
+		if confirm != "YES" {
```
Comment (isacikgoz) [inline commands/user.go:352]: I'm probably not going to miss yelling `YES/NO` options after the `v6` 🤓

File: commands/user.go (hunk 1 above)
Comment (mgdelacroix) [reply commands/user.go:88]: I'm really 0/5 on this, just used `deleteall` for consistency with the legacy CLI

Comment (isacikgoz) [reply commands/user.go:88]: Oh, I see 👍

**Pull request merged**